resources/configurationFiles/analysisIndelCalling.xml

```diff
@@ -53,6 +53,10 @@
         <cvalue name="LOCAL_CONTROL_COMMON_SNV" value="${hg19DatabasesDirectory}/LocalControls/pcawg_AF.All.CommonArtifacts.vcf.gz" type="path"/>
         <cvalue name="LOCAL_CONTROL_COMMON_SNV_2" value="${hg19DatabasesDirectory}/LocalControls/LocalControlConsolidated_AF.INFO.ACgt10.vcf.gz" type="path"/>
         <cvalue name="EXOME_CAPTURE_KIT_BEDFILE" value='${hg19DatabasesDirectory}/../targetRegions/Agilent5withUTRs_plain.bed.gz' type="path"/>
+        <!-- New gnomAD files and local control, needs to be moved to NGS share -->
+        <cvalue name="GNOMAD_V2_1_GENOME_SNV_INDEL" value="/icgc/dkfzlsdf/analysis/UniExome/GermlineAnnotation/annotationInfo/gnomAD/v2.1/genomes/gnomad.genomes.r2.1.sites.SNV-INDEL.vcf.gz" type="path" />
```

Comment:
These files should go to the `<filename>` section.

Reply:
These are database files required for the workflow, they are not workflow output. Should they still go to filename section?

Reply:
Oh, o.k., so they are kind of reference files. Sorry, leave them here.